code/modules/mob/living/basic/jungle/venus_human_trap.dm

```diff
@@ -173,9 +171,11 @@
 
 /mob/living/basic/venus_human_trap/Initialize(mapload)
 	. = ..()
-	var/datum/action/cooldown/vine_tangle/tangle = new(src)
-	tangle.Grant(src)
-	ai_controller.set_blackboard_key(BB_TARGETED_ACTION, tangle)
+	AddElement(/datum/element/lifesteal, 5)
```

Comment:
document change in changelog

Reply:
fixed my bad, also forgot to mention its a port fix